rest/route/aws.go

```diff
@@ -183,6 +183,12 @@ func (aws *awsSns) handleInstanceInterruptionWarning(ctx context.Context, instan
 		return nil
 	}
 
+	grip.Info(message.Fields{
+		"message": "got interruption warning from AWS",
+		"distro":  h.Distro,
```

Comment:
it might be a lot to log the entire distro document.
If you agree, I would think the relevant fields are the distro id and instance type.